server/koppsApi.js

```diff
@@ -99,6 +99,25 @@ function isDateInFuture(checkDate) {
   return false
 }
 
+async function getApplicationFromLadokUID(ladokUID) {
+  const { client } = api.koppsApi
+  const uri = `${config.koppsApi.basePath}courses/offerings/roundnumber?ladokuid=${ladokUID}`
+  log.info('Trying fetch courses application by', { ladokuid: ladokUID, uri, config: config.koppsApi })
+  try {
+    const res = await client.getAsync({ uri, useCache: true })
```

Comment:
it is better to extract body directly here like,  const { body } = await client.getAsync({ uri, useCache: true }) it counts as best practices. It is exactly the same just in one line.

Reply:
Done.